.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: /usr/src/Slicer-LiverAnalysis
+    docker:
+      - image: olevs/slicer-alive-build
+    steps:
+      - checkout
+      - run:
+          name: Update 3D Slicer
+          command: |
+            cd /usr/src/Slicer
+            git checkout .
+            git pull
```

Comment:
If we use the [slicer/slicer-build](https://hub.docker.com/r/slicer/slicer-build) image, here we would do the git clone, etc..

Reply:
Using aliveresearch/slicer-build seems to work